README.md

```diff
@@ -1,18 +1,58 @@
-# AOSC - Website
+# AOSC - Official Website
 
-Site oficial da Angola Open-source Community
+Project Overview - Angola Open Source Community Website
 
-## Getting Started
+This is the main README of the project for Open Source Contributions. The project is developed using React and JavaScript and is accompanied by READMEs in various languages, including:
```

Comment:
Os devs sabem que isso é um README xD. Não tem necessidade de mencionar.

Atenção que o projecto é website oficial da AOSC, e acho incoveniente mencionar `.. of the project for Open Source Contribuitions`